ably/realtime/realtime_channel.py

```diff
@@ -104,6 +164,22 @@ async def detach(self):
         self.set_state(ChannelState.DETACHED)
 
     async def subscribe(self, *args):
+        """Subscribe to a channel
+
+        Registers a listener for messages on the channel.
+
+        Parameters
+        ----------
+        *args: event, listener, optional
```

Comment:
is there a way we can list the different signatures and what they do?

for example:
```
*args: event, listener
  Subscribe to messages with the given event name

*args: listener
  Subscribe to all messages on the channel
```

also these don't match the descriptions from [here](https://github.com/ably/sdk-api-reference/blob/main/descriptions.md#class-channelschanneltype)